app/poros/npc.rb

```diff
@@ -117,6 +122,8 @@ def determine_ac(armor, piece, dex)
       16
     elsif piece.armor_class == '16' && armor.index(piece) == 1
       18
+    elsif piece.name == 'Shield'
+      2
```

Comment:
Before this, Clerics with Scale Mail and a Shield and a Dex 2 or higher would end up with AC of 20, because dex was being added to the sheild's AC too.